src/index.js

```diff
@@ -161,29 +165,96 @@ module.exports = function ({ types: t }) {
     if (node.__coverage__instrumented) return
     node.__coverage__instrumented = true
 
-    const id = nextStatementId(context, node.loc)
+    const id = nextStatementId(context, path)
     instrument(path, increase(context, 's', id))
   }
 
   //
+  // check a path to see if it has a ignore comment
+  // set the zone
+  //
+  function checkIgnoredPath (path, data) {
+    path.node.leadingComments && path.node.leadingComments.some((comment) => {
```

Comment:
On second thought, using `Array#some` is fine. Probably better than importing the whole babel-runtime thing.